src/docs/cursor-log.dox

```diff
@@ -31,12 +31,16 @@ uint32_t
 int
 @m_endif
 operation counter within that log record.  When a log record maps
-one-to-one to a transaction (in other words, the returned log record has
-the only database operation the transaction made), the operation counter
-returned for the key will be zero.
+to a record that is not a transaction, the operation counter
+returned for the key will be zero. When the log record maps to a
+transaction, the first record returned, with an operation counter of
+zero, will be the entire record. Then the WT_CURSOR::next call will
+step into the transaction and return the first individual operation
+within that transaction. So, a transaction with a single operation will
+return two records related to that transaction.
```

Comment:
I don't understand "a record that is not a transaction". Should this be "When a log record maps to a record that is not `part of` a transaction"?

Are we talking about records inserted outside of WiredTiger's transactional mechanisms? If an update is wrapped in a transaction by WiredTiger, rather than an explicit begin/commit in the application, then the log records are still part of a transaction. So, I'm confused.

Reply:
Should we note the operation counter is then incremented for each individual record?

Reply:
"a record that is not a transaction" refers to a log record that is not a transaction, not a key/value pair. So something like a log_printf message log record, or a checkpoint log record, etc. I will add the word `log` in front of each lone `record` for clarity.